lib/charms/layer/kubernetes_common.py

```diff
@@ -586,6 +586,8 @@ def generate_openstack_cloud_config():
         lines.append("floating-network-id = {}".format(openstack.floating_network_id))
     if openstack.lb_method:
         lines.append("lb-method = {}".format(openstack.lb_method))
+    if openstack.internal_lb:
+        lines.append("internal-lb = true")
```

Comment:
the description says it reads the config from the openstack-integrator relation -- but -- this just looks like static assignment in the config file.  I'm guessing there's some unfinished work here?  Perhaps this when the interface is merged?

```suggestion
    if openstack.internal_lb:
        lines.append("internal-lb = {}".format(openstack.internal_lb)
```

Reply:
`openstack.internal_lb` is defined in the interface, and it's a boolean option so the resulting `internal-lb` option in the configure is either present and set to true, or absent and defaulting to false.

Reply:
wow, it was an early morning.  sorry i totally missed this

Reply:
No, it was a valid point - it may in fact be better to always explicitly set this option.  ie. remove the conditional and use `lines.append("internal-lb = {}".format(openstack.internal_lb)`?